src/App.jsx

```diff
@@ -0,0 +1,82 @@
+import './App.css';
+import Card from './components/card';
+import Logo from './Images/logo.png';
+import Cart from './Images/cart.png';
+import Sosedi from './Images/sosedi.png';
+import Evroopt from './Images/evroopt.png';
+import Green from './Images/green.png';
+import Emall from './Images/emall.png';
+import Vitalur from './Images/vitalur.png';
+import Belmarket from './Images/belmarket.png';
+import Kopeechka from './Images/kopeechka.png';
+import Santa from './Images/santa.png';
+import Gippo from './Images/gippo.png';
```

Comment:
You can create here one helper file like `imges.jsx` and after import every thing from this file